packages/esbuild-plugin/README.md

```diff
@@ -50,24 +50,105 @@ require("esbuild").build({
+| urlSuffix          | `string`                                                                   | optional | URL suffix to add to the end of all filenames. Useful for appending query parameters.                                                                                                                                                                                                                                                                                                                                                                                                                                                                         |
+| validate           | `boolean`                                                                  | optional | When `true`, attempts source map validation before upload if rewriting is not enabled. It will spot a variety of issues with source maps and cancel the upload if any are found. Defaults to `false` as this can cause false positives.                                                                                                                                                                                                                                                                                                                       |
+| stripPrefix        | `array`                                                                    | optional | When paired with the `rewrite`, this will remove a prefix from filename references inside of sourcemaps. For instance you can use this to remove a path that is build machine specific. Note that this will NOT change the names of uploaded files.                                                                                                                                                                                                                                                                                                           |
+| stripCommonPrefix  | `boolean`                                                                  | optional | When paired with `rewrite`, this will add `~` to the `stripPrefix` array.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
```

Comment:
l: Let's either go with this suggestion or change my suggestion above (for `stripPrefix`) to remove the "the". Just to be consitent with the phrase
```suggestion
| stripCommonPrefix  | `boolean`                                                                  | optional | When paired with the `rewrite` option, this will add `~` to the `stripPrefix` array.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
```